src/VersionChip/VersionChip.test.tsx

```diff
@@ -29,6 +29,8 @@ describe("VersionChip", () => {
   });
   // check nothing is rendered if version format is wrong
   it("doesn't render when version is wrong format", () => {
+    // suppress expected console warning during test for invalid version format
+    vi.spyOn(console, "warn").mockImplementation(() => {});
```

Comment:
The ticket suggests to introduce `https://www.npmjs.com/package/vitest-fail-on-console` to avoid us introducing future warnings. 

As well as mocking this console, it would be good to add an `expect()` to prove that we got the console warning that we wanted.